build-tools/packages/build-tools/src/index.ts

```diff
@@ -5,11 +5,21 @@
 
 export { Context } from "./common/context";
 export { GitRepo } from "./common/gitRepo";
-export { FluidRepo, type VersionDetails } from "./common/fluidRepo";
+export {
+	type BrokenCompatTypes,
```

Comment:
I generally don't see a reason to include `type` here. Unless there is both a type and non-type value, it does the same thing. I only use import/export type explicitly when I want to ensure that the non-type value (if there is one) isn't used, usually because using it would cause an initialization time cyclic dependency issue. Nothing wrong with including it though I just haven't see this done this way.

Reply:
I like it because it's explicit about the intended use of the imported item. If it's only used as a type, import it as a type (IMO). It's also clearer that it will be completely removed from the JS and should have zero runtime impact. But I think if I really cared about that, I would push to enable [the lint rule](https://typescript-eslint.io/rules/consistent-type-imports/).